_drafts/2023-12-02-teorema-cap.md

```diff
@@ -185,17 +185,17 @@ Os sistemas alcançam a disponibilidade dos dados normalmente através da replic
 
 ## Partition Tolerance / Tolerância a Partições (P)
 
-O nível de **Tolerância a Partições** refere-se à capacidade de um banco de dados distribuído continuar operacional, apesar de uma ou mais de suas "partições" estarem indisponíveis ou no caso de eventuais falhas de comunicação entre seus nodes.
+O nível de **Tolerância a Partições** refere-se à capacidade de um banco de dados distribuído continuar operacional, apesar de falhas que "particionem" a rede e duas ou mais partes que não conseguem mais se comunicar.
 
 Em ambientes distribuídos, é cada vez mais necessário assumir que falhas de rede, hardware, atualizações programadas e de segurança emergencial feitas pelos provedores podem ocorrer a qualquer momento. Um sistema tolerante a partições é capaz de oferecer um certo nível de continuidade do serviço em caso de falhas parciais.
 
 Esse atributo é particularmente valioso em aplicações distribuídas geograficamente, redes sociais, agregadores de logs, brokers de eventos, sistemas de filas, entre outros.
 
-### O que é uma Partição de Dados?
+### O que é uma Partição de Rede?
```

Comment:
Eu não sei se fui eu que entendi errado, mas em se tratando de CAP o P geralmente se refere a partição de rede, não de dados.

Partição de dados eu entendo mais como dividir uma tabela entre vários bancos? Tipo assim: https://www.cockroachlabs.com/blog/what-is-data-partitioning-and-how-to-do-it-right/